src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE_FOR_FIGURES = 26;
+    private static final int MAX_VALUE_FOR_CIRCLE_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private Figure getRandomSquare() {
+        return new Square(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_VALUE_FOR_FIGURES));
+    }
+
+    private Figure getRandomRectangle() {
+        return new Rectangle(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_VALUE_FOR_FIGURES),
+                random.nextInt(MAX_VALUE_FOR_FIGURES));
+    }
+
+    private Figure getRandomRightTriangle() {
+        return new RightTriangle(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_VALUE_FOR_FIGURES),
+                random.nextInt(MAX_VALUE_FOR_FIGURES));
+
+    }
+
+    private Figure getRandomIsoscelesTrapezoid() {
+        return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_VALUE_FOR_FIGURES),
+                random.nextInt(MAX_VALUE_FOR_FIGURES),
+                random.nextInt(MAX_VALUE_FOR_FIGURES),
+                random.nextInt(MAX_VALUE_FOR_FIGURES));
+
+    }
+
+    private Figure getRandomCircle() {
+        return new Circle(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_VALUE_FOR_FIGURES));
+
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE, MAX_VALUE_FOR_CIRCLE_RADIUS);
+    }
+
+    public void getRandomFigure() {
```

Comment:
This method should return a Figure
```suggestion
    public Figure getRandomFigure() {
```